docs/release-notes.asciidoc

```diff
@@ -83,6 +71,8 @@ See ({pull}77327[#77327]) for details.
 [role="screenshot"]
 image::images/detection-rule-failure.png[]
 
+* When adding a rule exception, you cannot select value lists of type `ip_range`. Lists of type `ip_range` will not appear in the **Add Exception** dropdown as possible values after selecting the is in list operator. ({pull}79511[#79511]).
```

Comment:
I would wrap "is in list" in tick marks ``

Reply:
@DonNateR this has been sitting for 5 days. Please get this small change in and merge so we can close this out.